main.nf

```diff
@@ -217,7 +224,7 @@ if(!params.bismark_index && params.fasta && params.aligner == 'bismark'){
                    saveAs: { params.saveReference ? it : null }, mode: 'copy'
 
         input:
-        file fasta from fasta
+        file fasta from fasta_1
```

Comment:
Can you please use names that are more explicit?

ch_from_blabla_to_blabla is something that helps you (and me, others) to keep things traceable :-)